pymc3/stats.py

```diff
@@ -166,17 +166,11 @@ def logp_vals_point(pt):
     except AttributeError:
         points = trace
 
-    points = tqdm(points) if progressbar else points
-
-    try:
-        logp = (logp_vals_point(pt) for pt in points)
-        return np.stack(logp)
-    finally:
-        if progressbar:
-            points.close()
+    logp = (logp_vals_point(pt) for pt in points)
+    return np.stack(logp)
 
 
-def waic(trace, model=None, pointwise=False, progressbar=False):
+def waic(trace, model=None, pointwise=False):
```

Comment:
not related with your PR, but given that you are already there, could you change "widely available information criterion" to "the widely applicable information criterion"? :-)